UsingGoogleSignInButton.md

```diff
@@ -0,0 +1,44 @@
+# Using the Google Sign In Button
+
+Depending upon your platform, GoogleSignIn-iOS provides a few options to use
+when creating a sign-in button.
+
+## SwiftUI
```

Comment:
What if we went with a UI framework focused structure?

## SwiftUI (iOS and macOS)
## UIKit (iOS)
## AppKit (macOS)

Reply:
Sure thing.